readme.md

```diff
@@ -31,7 +31,7 @@ This is possible because [we use the Parcel library](https://en.parceljs.org/scs
 ## Checklist
 
 ❗️ Replace `<your_account>` with your GitHub username and copy the links to the `Pull Request` description:
-- [DEMO LINK](https://<your_account>.github.io/layout_stop-watch/)
+- [DEMO LINK](https://GabrielNunes-383.github.io/layout_stop-watch/)
```

Comment:
Seems to me that the hour pointer is moving before we complete the hole circle with the minutes arrow passing the hour arrow. This is happening a little bit sooner